prime-router/src/main/kotlin/Element.kt

```diff
@@ -207,6 +220,13 @@ data class Element(
                         ?: error("Invalid Code: '$formattedValue' does not match any codes for '${name}'")
                 }
             }
+            Type.TELEPHONE -> {
+                val number = phoneNumberUtil.parse(formattedValue, "US")
+                if (!number.hasNationalNumber() || number.nationalNumber > 9999999999L)
+                    error("Invalid phone number '$formattedValue' for '$name'")
+                val nationalNumber = DecimalFormat("0000000000").format(number.nationalNumber)
+                "${nationalNumber}$phoneDelimiter${number.countryCode}$phoneDelimiter${number.extension}"
+            }
```

Comment:
Always add all 3 parts.

Reply:
Is the "+" was a canonical part of country code?  If so, are we handling that?